reposcan/database/upgrade.py

```diff
@@ -36,6 +36,8 @@ class DatabaseUpgrade:
     def __init__(self):
         LOGGER.info('DatabaseUpgrade initializing.')
 
+        DatabaseHandler.close_connection()
```

Comment:
Here it is closing connection already in constructor, if there is some remaining connection from previous stuff. This made problems in tests.